test/e2e/applicationset/applicationset_test.go

```diff
@@ -508,3 +508,72 @@ func TestCustomApplicationFinalizers(t *testing.T) {
 		When().
 		Delete().Then().Expect(ApplicationsDoNotExist([]argov1alpha1.Application{expectedApp}))
 }
+
+func TestSimplePullRequestGenerator(t *testing.T) {
+	expectedApp := argov1alpha1.Application{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Application",
+			APIVersion: "argoproj.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:       "guestbook-2",
+			Namespace:  utils.ArgoCDNamespace,
+			Finalizers: []string{"resources-finalizer.argocd.argoproj.io"},
+		},
+		Spec: argov1alpha1.ApplicationSpec{
+			Project: "default",
+			Source: argov1alpha1.ApplicationSource{
+				RepoURL:        "git@github.com:shmurata/argocd-example-apps.git",
+				TargetRevision: "824a5c987fdfb2b0629e9dbf5f31636c69ba4772",
+				Path:           "kustomize-guestbook",
+				Kustomize: &argov1alpha1.ApplicationSourceKustomize{
+					NamePrefix: "guestbook-2",
+				},
+			},
+			Destination: argov1alpha1.ApplicationDestination{
+				Server:    "https://kubernetes.default.svc",
+				Namespace: "guestbook-pull-request",
+			},
+		},
+	}
+
+	Given(t).
+		// Create an PullRequestGenerator-based ApplicationSet
+		When().Create(v1alpha1.ApplicationSet{ObjectMeta: metav1.ObjectMeta{
+		Name: "simple-pull-request-generator",
+	},
+		Spec: v1alpha1.ApplicationSetSpec{
+			Template: v1alpha1.ApplicationSetTemplate{
+				ApplicationSetTemplateMeta: v1alpha1.ApplicationSetTemplateMeta{Name: "guestbook-{{ number }}"},
+				Spec: argov1alpha1.ApplicationSpec{
+					Project: "default",
+					Source: argov1alpha1.ApplicationSource{
+						RepoURL:        "git@github.com:shmurata/argocd-example-apps.git",
```

Comment:
I've created a new test org for ApplicationSet repo to uses for test like this:
https://github.com/applicationset-test-org/argocd-example-apps

I've copied over the contents of your repository (the branches, and the PR) to this new org. Are you able to update the test to use this new org? Take a look and let me know if I missed something :sweat_smile:

Reply:
Thank you for preparing the repository.
I would like to ask for one additional setting.
In this test, we are also checking the label filter. Can you please add a label to the pull request that you prepared?

Reply:
@shmurata Done.

Reply:
Thanks. Fix in 6bbe516.